RELEASE_NOTES.md

```diff
@@ -3,9 +3,11 @@
 ## in dev
 
 * DxNI handles native app(let)s with optional non-file object inputs
+* Adds support for Amazon ECR repositories
```

Comment:
Thanks. Please add info about the change in Extras (the new fields) and in the scatter chunk.

Reply:
done